extract_firmware.py

```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
Is this necessary? If there's no py2, python will be python3, right? And the script should work either way...